src/openforms/analytics_tools/models.py

```diff
@@ -197,6 +205,33 @@ class AnalyticsToolsConfiguration(SingletonModel):
         help_text=_("Enabling this installs SiteImprove"),
     )
 
+    govmetric_source_id = models.CharField(
+        _("GovMetric source ID"),
+        max_length=10,
+        blank=True,
+        help_text=_(
+            "Your GovMetric source ID - This is created by KLANTINFOCUS when a list of questions is created. "
+            "It is a numerical value that is unique per set of questions."
+        ),
+    )
+    govmetric_secure_guid = models.CharField(
+        _("GovMetric secure GUID"),
+        blank=True,
+        max_length=10,
```

Comment:
I don't know the real structure of the ID on the GovMetric side, but GUIDs (which in most cases are similar/identical to UUIDs) are more than 10 characters long?